ionic.config.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Please don't check in this file - it is specific to my compiled version in the app store. Other installations should have their own app id.

Reply:
you can remove it using `git rm --cached ionic.config.json`